package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": "0.1.0",
+  "name": "com.bluefletch.motorola",
+  "cordova_name": "com.bluefletch.motorola",
+  "description": "Motorola Cordova Plugin",
+  "license": "MIT",
+  "repo": "https://github.com/RsanabriaZ/flq-cordova-plugin-laserscanner.git",
```

Comment:
Don't use your public repo here. If this repo has been forked over to the foodlogiq organization, then please update this line to use the flq repo.